teamcity/flake8_plugin.py

```diff
@@ -1,74 +1,28 @@
-try:
-    import pycodestyle as pep8
-except ImportError:
-    import pep8
+from flake8.formatting import base
 
 import re
 
 from teamcity.messages import TeamcityServiceMessages
-from teamcity import __version__, is_running_under_teamcity
```

Comment:
me too. Anyway, I definitely prefer the experience without specifying explicit formatter because it just works, no need to change your scripts other than installing teamcity-messages